CustomerContactBook/Services/GroupService.cs

```diff
@@ -0,0 +1,107 @@
+﻿using CustomerContactBook.Database;
+using CustomerContactBook.Database.Tables;
+using CustomerContactBook.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace CustomerContactBook.Services
+{
+    public class GroupService
+    {
+        private readonly ContactBookContext _context;
+
+        public GroupService(ContactBookContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<List<CustomerGroupModel>> GetGroups()
```

Comment:
in the other methods you use `CustomerGroup`, maybe add `Customer` word here as well?